indy_common/authorize/auth_constraints.py

```diff
@@ -20,18 +24,30 @@ def __init__(self, role, sig_count, need_to_be_owner=False, metadata={}):
         self.metadata = metadata
         self.constraint_id = ROLE_CONSTRAINT_ID
 
+    def __str__(self):
+        role = get_named_role(self.role) if self.role != '*' else 'ALL'
+        return "Required role: {}, Count of signatures: {}, Need to be owner: {}".format(role,
```

Comment:
Maybe we should have shorter description.
Cases:
- owner is None and role != *:
    `X <role> signatures are required `
- owner is None and role == *:
    `X signatures of any role are required`
- owner is not None and number  of sigs = 1:
    `<role> needs to be owner`
- owner is not None, number of sigs = 1, and role == *
    `needs to be owner`
- owner is not None and number  of sigs = 1 and role != *
    `X <role> signatures are required and needs to be owner`
- owner is not None and number  of sigs = 1 and role == *
    `X signatures of any role are required and needs to be owner`
- else (fallback):
    as in the PR